MerlinAU.sh

```diff
@@ -4853,7 +5447,20 @@ _RunOfflineUpdateNow_()
             printf "\nContinuing to the update file selection process.\n"
             if _SelectOfflineUpdateFile_
             then
-                set -- $(_GetLatestFWUpdateVersionFromWebsite_ "$FW_URL_RELEASE")
+                if "$isGNUtonFW"
+                then
+                    # Extract the filename from the path
+                    original_filename=$(basename "$selected_file")
+                    # Sanitize filename by removing problematic characters (if necessary)
+                    sanitized_filename=$(echo "$original_filename" | sed 's/[^a-zA-Z0-9._-]//g')
+                    # Extract the file extension
+                    extension="${sanitized_filename##*.}"
+                    FW_DL_FPATH="${FW_ZIP_DIR}/${FW_FileName}.${extension}"
+                    _GnutonBuildSelection_
+                    set -- $(_GetLatestFWUpdateVersionFromGithub_ "$FW_GITURL_RELEASE" "$firmware_choice")
+                else
+                    set -- $(_GetLatestFWUpdateVersionFromWebsite_ "$FW_URL_RELEASE")
```

Comment:
Started taking a quick look & noticed the changes in the var names for the URLs so I ran a search and found this error in line **5462**: 

The var name "**FW_URL_RELEASE**" should be "**FW_SFURL_RELEASE**"

Reply:
@Martinski4GitHub 

Great catch!!! Corrected :)